src/MainWindow.cpp

```diff
@@ -152,10 +138,8 @@ MainWindow::MainWindow(BRect frame) : BWindow(frame, "", B_DOCUMENT_WINDOW, 0)
 
 	menu = new BMenu(B_TRANSLATE("Tools"));
 	bar->AddItem(menu);
-	temp = B_TRANSLATE("Budget…");
-	menu->AddItem(new BMenuItem(temp.String(), new BMessage(M_SHOW_BUDGET_WINDOW)));
-	temp = B_TRANSLATE("Reports…");
-	menu->AddItem(new BMenuItem(temp.String(), new BMessage(M_SHOW_REPORTS_WINDOW)));
+	menu->AddItem(new BMenuItem(B_TRANSLATE("Budget…"), new BMessage(M_SHOW_BUDGET_WINDOW)));
```

Comment:
Can't totall explain it, but I think these 2 items would work without "…". Those two tools are kinda separate from the main window, not just some 2ndary window you do an edit and return to the main window.
There are so many items with "…" already... Your call. :)